docs/src/documentation/03-components/07-interaction/stepper/03-accessibility.mdx

```diff
@@ -16,7 +16,13 @@ The following props provide additional information to screen readers:
 
 - The `titleIncrement` prop allows you to specify an `aria-label` attribute for the increment icon button in the Stepper (StepperStateless) component.
 
-- The `ariaLabelledBy` prop allows you to specify an `aria-labelledby` attribute for the Stepper component. This attribute references the ID of the element that labels the Stepper (StepperStateless), ensuring that screen readers announce the label correctly.
+- The `descriptionDecrement` prop allows you to specify an `aria-describedby` attribute for the decrement icon button in the Stepper (StepperStateless) component.
+
+- The `descriptionIncrement` prop allows you to specify an `aria-describedby` attribute for the increment icon button in the Stepper (StepperStateless) component.
+
+- The `ariaLabelledBy` prop allows you to specify an `aria-labelledby` attribute for the input field in the Stepper component. This attribute references the ID of the element that labels the Stepper (StepperStateless), ensuring that screen readers announce the label correctly.
+
+- The `ariaDescribedby` prop allows you to specify an `aria-describedby` attribute for the input field in the Stepper component. This attribute references the ID of the element that describes the Stepper (StepperStateless), providing additional context or instructions.
```

Comment:
I would group here props for input vs prop for buttons so it's easier for the reader to make sense of it.

And I would unify the information, e.g. for `ariaDescribedby` you mention what the attribute references but you don't do it for `descriptionDecrement`. Sometimes input element is mentioned, sometimes input field.

Would it make sense to maybe put these props in the table and write some common text under the group of elements?